src/components/UserProfile/helper.ts

```diff
@@ -0,0 +1,37 @@
+import { ProjectStatus, TaskStatus, User } from 'utils/types';
+//handle comvert status, member
+export const convertTaskStatus = (status: TaskStatus): string => {
+   switch (status) {
+      case 1:
+         return TaskStatus[1];
```

Comment:
```suggestion
         return TaskStatus.Requesting;
```

Tương tự với cách compare khác của TaskStatus và ProjectStatus, chú ý phần này: enum nên dùng key của enum, k dùng index

Reply:
Cái này em thấy có vẻ em viết thừa ạ, nên em bỏ đi luôn ạ